### ISSUE TITLE
Pseudo-selectors?

### ISSUE THREAD

**Issue by mexn** [giuseppeg/postcss-pseudo-classes#10]
Hi,
I really like this package, we recently integrated it for generating a state table, showing links and inputs in various states. I stumbled upon an article about considering contrast for various scenarios (https://css-tricks.com/a-whole-bunch-of-places-to-consider-contrast-in-a-single-paragraph/) lately, so I tried to integrate `::selection` in this state table, too, using your package. As this did not work, I checked the docs, source code and pull requests, so I'm now aware, that pseudo-selectors are excluded on purpose (I probably could have also guessed it from the name 🙄 ), but I am curious about the background, why this decision was made and if you could consider some exceptions, like mentioned `::selection`, or e.g. `::placeholder`?

**Comment by giuseppeg** [comment]
Hi Markus, thank you for the kind words and reaching out! 

> pseudo-selectors are excluded on purpose
> I am curious about the background, why this decision was made and if you could consider some exceptions

This plugin is based on https://github.com/SlexAxton/rework-pseudo-classes 

Initially I simply ported it to PostCSS and over time I fixed a few things and added a few corner cases. To be honest I never thought about adding pseudo-elements (including before and after), and I don't use it right now. 

Probably your best option would be to fork it and add the exception to cover your use case
